libs/orpc/src/dex-gateway.type.ts

```diff
@@ -48,6 +48,19 @@ export interface SwapRequest {
   tracking_id: string;
 }
 
+export interface AddLiquidityRequest {
```

Comment:
I would ue another module to avoid confusion around types used with gateway and those used on the chain